### ISSUE TITLE
PostgreSQL support

### ISSUE THREAD

**Issue by le0pard** [khiav223577/find_with_order#5]
Hello. Looks like this gem will not work in PostgreSQL, only MySQL (no support for `ORDER BY FIELD()`). You can use something like this for PostgreSQL:

```sql
SELECT foo.* FROM foo                                           
JOIN (SELECT id.val, row_number() over() FROM (VALUES(3),(2),(6),(1),(4)) AS
id(val)) AS id
ON (foo.catalog_id = id.val) ORDER BY row_number;
```

example:

```sql
$ SELECT * FROM users WHERE id IN (3,2,1) ORDER BY FIELD(id, 3,2,1);
ERROR:  function field(integer, integer, integer, integer) does not exist
LINE 1: SELECT * FROM users WHERE id IN (3,2,1) ORDER BY FIELD(id, 3...
                                                         ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.

$ SELECT users.* FROM users JOIN (SELECT id.val, row_number() over() FROM (VALUES(3),(2),(1)) AS id(val)) AS id ON (users.id = id.val) ORDER BY row_number;

id |      name
----+-----------------
  2 |
  1 | Alexey Vasiliev
(2 rows)
```

**Comment by khiav223577** [comment]
Thanks for the hint :) 
I'll take a look at it soon.
Or would you mind submitting a PR?

**Comment by le0pard** [comment]
@khiav223577 sorry, don't have time right now :(

**Comment by khiav223577** [comment]
It's OK. Thanks for your hint again :)

**Comment by khiav223577** [comment]
Just submitted a PR for this. - https://github.com/khiav223577/find_with_order/pull/7

**Comment by khiav223577** [comment]
`1.1.0` was released. It supports PostgreSQL now.

**Comment by darcang** [comment]
@khiav223577 Hey, is there any solution to support pgsql uuid type?

**Comment by khiav223577** [comment]
@darcang 
What is your error message of using `uuid `type?
`operator does not exist: uuid = text`?

**Comment by darcang** [comment]
@khiav223577 yeah, i got this working for PG >= 9.5 using
```ruby
User.where(id: ids)
  .order(Arel.sql("array_position(ARRAY['#{ids.join("','")}']::uuid[], users.id)"))

```

**Comment by khiav223577** [comment]
Hi, @darcang 
I released [1.3.0](https://rubygems.org/gems/find_with_order/versions/1.3.0) to support psql uuid type.
See the implementation here: #18

**Comment by darcang** [comment]
@khiav223577 nice, great thx!
